docs/source/peer_ledger_snapshot.md

```diff
@@ -0,0 +1,153 @@
+
+## Overview
+
+Snapshots can be used by organizations that already have peers on a channel or by organizations new to a channel. Whatever the use case, the process is largely the same.
+
+If the organization is new to the channel, the organizations currently joined to the channel must:
+
+1. **Schedule a snapshot**. These snapshots must be taken at **exactly the same ledger height** on each peer. This will allow the new peer organization to evaluate the snapshots to make sure they contain the same data. This ledger height must be equal or higher than the current ledger height. They cannot be taken from a lower ledger height. If you attempt to schedule a snapshot at a height lower than the current height you will get an error. Note that a peer that already used a snapshot to join a channel can also be used to take a snapshot.
+2. **When the ledger height is reached, the snapshot is taken by the peer**. The snapshot is comprised of folders that include the public state, hashes of private state, transaction IDs, and the collection config history. A file containing metadata relating to these files is also included. For more information, check out [contents of a snapshot](#contents-of-a-snapshot).
+2. **The snapshot is sent to the new organization**. This must be completed out of band. Because snapshot files are not compressed, it is likely that peer administrators will want to compress these files before sending them.
+
+The new organization will then:
+
+1. **Evaluate the snapshots from the other organizations to ensure they contain the same data**. An administrator of the peer organization attempting to use the snapshot to join the peer to the channel should independently compute the hashes of the snapshot files and match these with the hashes present in the metadata file.
+2. **Join the peer to the channel using the snapshot**. When the peer has finished joining the channel using the snapshot, it will begin pulling private data according to the collections it member of.
+3. **Verify the peer has been join the channel successfully**. For more information, check out [Joining a channel using a snapshot](#joining-a-channel-using-a-snapshot).
+
+If an organization that is already joined to the channel wants to join a new peer using a snapshot, it might decide to skip the process of having other organizations take snapshots and evaluate them, though it is a best practice for an organization to periodically take snapshots of all its peers and compare them to ensure the no ledger forks have occurred. In that case, the organization can take a snapshot immediately and then use the snapshot to join the new peer to the channel.
+
+**Note: snapshots can also be used to verify that the public state between peers is identical, even if no new peer will use the snapshot to join the channel. This can be useful in identifying if a ledger fork has occurred.**
```

Comment:
Would it add value to put something along the lines of how snapshots can help identify a ledger fork?  Is this referring to commit hashes, or would other meta data be used, etc?

Reply:
Yes, indeed. We can add here that the value of the field `snapshot_hash` in the file `_snapshot_additional_metadata.json` in the snapshots generated across peers should be the same.